src/main/java/com/petitapetit/miml/domain/playlist/controller/PlayListController.java

```diff
@@ -1,19 +1,67 @@
 package com.petitapetit.miml.domain.playlist.controller;
 
 import com.petitapetit.miml.domain.playlist.dto.PlayListDto;
+import com.petitapetit.miml.domain.playlist.service.PlayListService;
+import com.petitapetit.miml.util.UriUtil;
+import lombok.RequiredArgsConstructor;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
 
 @RestController
 @RequestMapping("/playlists")
+@RequiredArgsConstructor
 public class PlayListController {
 
+    private final PlayListService playListService;
+    public static final String DEFAULT_URI = "/playlists";
+
+
     @PostMapping
-    public ResponseEntity postPlayList(@RequestBody PlayListDto.Post requestBody){
+    public ResponseEntity<PlayListDto.SaveResponse> postPlayList(@RequestBody PlayListDto.SaveRequest request){
+
+        Long memberId = 1L; //mock
+        PlayListDto.SaveResponse response = playListService.savePlayList(request , memberId);
+
+        URI uri = UriUtil.createUri(DEFAULT_URI, response.getPlayListId());
+
+        return ResponseEntity.created(uri).body(response);
+    }
+
+    @PatchMapping("/{play_list_id}")
+    public ResponseEntity<PlayListDto.SaveResponse> patchPlayList(@PathVariable("play_list_id") Long playListId,
+                                                                  @RequestBody PlayListDto.SaveRequest request){
+
+        Long memberId = 1L;//mock
+        PlayListDto.SaveResponse response = playListService.modifyPlayList(playListId, request, memberId);
+
+        return ResponseEntity.ok(response);
+    }
```

Comment:
`NIT` : 요건 단순한 팁인데 단순히 `JSON` + `HttpStatus.OK` 라면 ResponseEntity 를 Wrapping 하지 않으셔도 됩니다. 일관성이 있어 보기 좋네요 😀
```suggestion
    @PatchMapping("/{play_list_id}")
    public PlayListDto.SaveResponse patchPlayList(@PathVariable("play_list_id") Long playListId,
                                                                  @RequestBody PlayListDto.SaveRequest request){

        Long memberId = 1L;//mock
        return playListService.modifyPlayList(playListId, request, memberId);
    }
```

Reply:
넵 참고하도록 하겠습니다 :)